deisctl/cmd/upgrade.go

```diff
@@ -65,16 +69,15 @@ func republishServices(ttl uint64, nodes []*model.ConfigNode, cb config.Backend)
 }
 
 // UpgradeTakeover gracefully starts a platform stopped with UpgradePrep
-func UpgradeTakeover(b backend.Backend, cb config.Backend) error {
-
-	if err := doUpgradeTakeOver(b, cb); err != nil {
+func UpgradeTakeover(stateless bool, b backend.Backend, cb config.Backend) error {
```

Comment:
It would be nice to have the ability to specify version upgrading to and validate unit files are desired version.